pkg/testutils/testutils.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package testutils
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/syzkaller/pkg/db"
+	"github.com/google/syzkaller/prog"
+)
+
+func ReadCorpus(filename string, target *prog.Target) []*prog.Prog {
+	if filename == "" {
+		return nil
+	}
+	db, err := db.Open(filename)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to open corpus database: %v\n", err)
```

Comment:
This needs to return an error and let caller handle it.
Exiting on errors is only fine for top level programs b/c they know what they are doing. But a function in a package should never terminate the whole program on errors.
Also e.g. not possible to test.